verify-replace.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+TMP_DIR=/tmp/
+BASE_REPO_PATH=$(mktemp -d ${TMP_DIR}replace-verify.XXX)
+GH_BASE_URL_KS=https://github.com/kubesaw/
+GH_BASE_URL_CRT=https://github.com/codeready-toolchain/
+GH_KSCTL=${GH_BASE_URL_KS}ksctl
+GH_HOST=${GH_BASE_URL_CRT}host-operator
+GH_MEMBER=${GH_BASE_URL_CRT}member-operator
+GH_REGSVC=${GH_BASE_URL_CRT}registration-service
+GH_E2E=${GH_BASE_URL_CRT}toolchain-e2e
+GH_TC=${GH_BASE_URL_CRT}toolchain-common
+C_PATH=${PWD}
+ERRORLIST=()
+
+echo Initiating verify-replace on dependent repos
+for repo in ${GH_HOST} ${GH_REGSVC} ${GH_KSCTL} ${GH_MEMBER} ${GH_E2E}
+do
+    echo =========================================================================================
+    echo  
+    echo $(basename $repo)
+    echo                                                                     
+    echo =========================================================================================                                            
+    REPO_PATH=$BASE_REPO_PATH/$(basename $repo)
+    echo Cloning repo in /tmp
+    git clone --depth=1 $repo $REPO_PATH
+    echo Repo cloned successfully
+    cd $REPO_PATH
```

Comment:
```suggestion
    cd $REPO_PATH || exit
```
or return? Incase cd fails?

Reply:
its a good point added it

Reply:
@ranakan19 could you please clarify why you would add exit?
The repo should be created by now, so not sure why cd would fail. And if it fails, then it would exit the script anyway.